seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/source/reader/TextReadStrategy.java

```diff
@@ -47,15 +47,19 @@ public class TextReadStrategy extends AbstractReadStrategy {
     private DateUtils.Formatter dateFormat = DateUtils.Formatter.YYYY_MM_DD;
     private DateTimeUtils.Formatter datetimeFormat = DateTimeUtils.Formatter.YYYY_MM_DD_HH_MM_SS;
     private TimeUtils.Formatter timeFormat = TimeUtils.Formatter.HH_MM_SS;
+    private Long skipHeaderNumber = 0L;
 
     @Override
     public void read(String path, Collector<SeaTunnelRow> output) throws IOException, FilePluginException {
+        if (pluginConfig.hasPath(BaseSourceConfig.SKIP_HEADER_ROW_NUMBER)) {
```

Comment:
Treat `skipHeaderNumber` as a class attribute and init it in `setPluginConfig` is better.

Reply:
I'll find a time to optimize

Reply:
OK, I resolved.